.github/workflows/install_requirements.yml

```diff
@@ -16,7 +16,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-18.04, ubuntu-20.04, windows-2019, macos-10.15, macos-11]
-        python: [3.6, 3.7, 3.8, 3.9]
+        python: [3.6, 3.7, 3.8, 3.9, "3.10"]
```

Comment:
Without quotes like the others?

Reply:
it does not work without quotes.
yaml parse numbers without quotes as floats, so instead of 3.10 we got 3.1 and workflow failed

Reply:
That's surprising.